nikola/plugins/basic_import.py

```diff
@@ -140,6 +140,24 @@ def write_content(cls, filename, content, rewrite_html=True):
         with open(filename, "wb+") as fd:
             fd.write(content)
 
+    @classmethod
```

Comment:
Out of curiosity: what's so bad about making something `@classmethod` in Python?